<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkFontsWidget</class>
 <widget class="QWidget" name="WkFontsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font families</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Standard font:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="StandardChooser_" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Monospace font:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="FixedChooser_" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Serif font:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="SerifChooser_" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sans-serif font:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="SansSerifChooser_" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Cursive font:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="CursiveChooser_" native="true"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Fantasy font:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LeechCraft::Util::FontChooserWidget" name="FantasyChooser_" native="true"/>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="ChangeAll_">
        <property name="text">
         <string>Change all...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Font sizes</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Default:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SizeDefault_">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="SizeMinimum_">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Fixed-width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="SizeFixedWidth_">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Font zoom:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="Zoom_">
        <property name="suffix">
         <string notr="true">%</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1600</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Util::FontChooserWidget</class>
   <extends>QWidget</extends>
   <header>fontchooserwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
